### PR TITLE
parallel 20200722 (downgrade)

### PR DESCRIPTION
**Pull request by tiiiecherle**
see https://github.com/Homebrew/homebrew-core/issues/60146

-----
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/parallel.rb [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
   head "https://git.savannah.gnu.org/git/parallel.git"
 
   bottle do
     cellar :any_skip_relocation
-    sha256 "5a78bb25a7ab4168c770d1bfef742afffa6c35229ca96ed0ad8135113267d99c" => :catalina
-    sha256 "3ba75a7522a91db40082095066738a93d7b92e24c1ed49815a528be475565eef" => :mojave
-    sha256 "dda2ab83abf9acb274d260ab9b2c0751f1a0e99c31a579f50dd6846e3eefc2de" => :high_sierra
+    sha256 "eff11ec3d9bf05b44b379c0993c5f3687f775b564864cd1557a1168a51ebad61" => :catalina
+    sha256 "eff11ec3d9bf05b44b379c0993c5f3687f775b564864cd1557a1168a51ebad61" => :mojave
+    sha256 "eff11ec3d9bf05b44b379c0993c5f3687f775b564864cd1557a1168a51ebad61" => :high_sierra
```
Comment (bayandin) [inline Formula/parallel.rb:14]: No need to change `bottle` block
```suggestion
    sha256 "5a78bb25a7ab4168c770d1bfef742afffa6c35229ca96ed0ad8135113267d99c" => :catalina
    sha256 "3ba75a7522a91db40082095066738a93d7b92e24c1ed49815a528be475565eef" => :mojave
    sha256 "dda2ab83abf9acb274d260ab9b2c0751f1a0e99c31a579f50dd6846e3eefc2de" => :high_sierra
```

File: Formula/parallel.rb [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
   desc "Shell command parallelization utility"
   homepage "https://savannah.gnu.org/projects/parallel/"
-  url "https://ftp.gnu.org/gnu/parallel/parallel-20200822.tar.bz2"
-  mirror "https://ftpmirror.gnu.org/parallel/parallel-20200822.tar.bz2"
-  sha256 "9654226a808392c365b1e7b8dea91bf4870bc4f306228d853eb700679e21be09"
-  license "GPL-3.0-or-later"
+  url "https://ftp.gnu.org/gnu/parallel/parallel-20200722.tar.bz2"
+  mirror "https://ftpmirror.gnu.org/parallel/parallel-20200722.tar.bz2"
+  sha256 "4801d44f2f71eed26386a0623a6fb3cadd7fa7ec2b5a7bbc5b7b52e2a0450d6f"
+  license "GPL-3.0"
```
Comment (bayandin) [inline Formula/parallel.rb:7]: Do not revert changes in license

```suggestion
  license "GPL-3.0-or-later"
```

File: Formula/parallel.rb [hunk 3]
```diff
@@ -1,17 +1,17 @@
 class Parallel < Formula
   desc "Shell command parallelization utility"
   homepage "https://savannah.gnu.org/projects/parallel/"
-  url "https://ftp.gnu.org/gnu/parallel/parallel-20200822.tar.bz2"
-  mirror "https://ftpmirror.gnu.org/parallel/parallel-20200822.tar.bz2"
-  sha256 "9654226a808392c365b1e7b8dea91bf4870bc4f306228d853eb700679e21be09"
-  license "GPL-3.0-or-later"
+  url "https://ftp.gnu.org/gnu/parallel/parallel-20200722.tar.bz2"
+  mirror "https://ftpmirror.gnu.org/parallel/parallel-20200722.tar.bz2"
```
Comment (bayandin) [inline Formula/parallel.rb:5]: Please add a comment that `20200822` is a beta release for Mac (with a link to release notes)

File: Formula/parallel.rb [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
+  sha256 "4801d44f2f71eed26386a0623a6fb3cadd7fa7ec2b5a7bbc5b7b52e2a0450d6f"
+  license "GPL-3.0"
   head "https://git.savannah.gnu.org/git/parallel.git"
```
Comment (bayandin) [inline Formula/parallel.rb:8]: Also, we need to bump `version_scheme` to make sure all the users will get the update

```suggestion
  version_scheme 1
  head "https://git.savannah.gnu.org/git/parallel.git"
```

**Review by bayandin — CHANGES_REQUESTED** [review]
And after all you need to squash all the commits into single one `parallel 20200722`

File: Formula/parallel.rb (hunk 3 above)
Comment (tiiiecherle) [reply Formula/parallel.rb:5]: @bayandin  hey, thanks for the changes, what do you mean by comment? Where shall it be added?

**Review by tiiiecherle — COMMENTED** [review]
thanks @bayandin

File: Formula/parallel.rb (hunk 3 above)
Comment (bayandin) [reply Formula/parallel.rb:5]: Please add a comment above a line with `url`:
```ruby
  # a comment
  url "https://ftp.gnu.org/gnu/parallel/parallel-20200722.tar.bz2"
```

Comment (tiiiecherle) [reply Formula/parallel.rb:7]: @bayandin done ;)

**Comment by bayandin** [comment]
> And after all you need to squash all the commits into single one `parallel 20200722`

And the last required thing

**Comment by SMillerDev** [comment]
@samford you might want to mark this in livecheck too.

**Comment by tiiiecherle** [comment]
> > And after all you need to squash all the commits into single one `parallel 20200722`
> 
> And the last required thing

@bayandin sorry, I don't know what you mean or how to do it. Could you please take this step and merge the commit afterwards. I will have a look at it later for the next time. Thanks

**Comment by tiiiecherle** [comment]
@bayandin Thanks. All checks passed and hopefully ready to go ;)

**Review by bayandin — APPROVED** [review]
Thank you @tiiiecherle

**Comment by tiiiecherle** [comment]
@bayandin The merge is somehow blocked. Thanks for having a look ;)

**Comment by SMillerDev** [comment]
CI will merge it.

**Pull request closed without merging**